### PR TITLE
Basic patches to add FreeBSD compilation to tree

### PR DESCRIPTION
**Pull request by DianeBruce**
Simple minor path changes.

### CODE REVIEW

File: cmake/Modules/FindQwt.cmake [hunk 1]
```diff
@@ -15,10 +15,10 @@ find_path(QWT_INCLUDE_DIRS
   ${CMAKE_INSTALL_PREFIX}/include/qwt
   ${CMAKE_PREFIX_PATH}/include/qwt
   PATHS
+  /usr/local/include/qt5/qwt6
   /usr/local/include/qwt-${QWT_QT_VERSION}
   /usr/local/include/qwt
   /usr/include/qwt6
-  /usr/include/qt5/qwt
```
Comment (nickoe) [inline cmake/Modules/FindQwt.cmake]: If you are only adding, why did you remove here?

**Comment by nickoe** [comment]
https://github.com/gnuradio/gnuradio/pull/2409/commits/6e24004151d795c2e747ebe25280568eb23c0155
I am not sure I understand the thing about dist-packages vs site-packages compared to debian vs everyone else. A commit was made to generalize this a while ago https://github.com/gnuradio/gnuradio/commit/0d6aa71a114a2d2c6be95bc59c45a621bf053c82
But this was reverted shortly after again in https://github.com/gnuradio/gnuradio/commit/9f5e65d8ae9608acf7526095787d37c95f8fd543

I am not exactly sure what the issue was with it that broke the CMAKE_INSTALL_PREFIX.
<hr>

https://github.com/gnuradio/gnuradio/pull/2409/commits/53ee6625782176915756a89bfc625d62d8bbad67
Your last changes are related the following pull request. Did you consider it?
https://github.com/gnuradio/gnuradio/pull/2293

File: cmake/Modules/FindQwt.cmake (hunk 1 above)
Comment (DianeBruce) [reply cmake/Modules/FindQwt.cmake]: Because I screwed up. Sorry :-(

**Comment by DianeBruce** [comment]
> [6e24004](https://github.com/gnuradio/gnuradio/commit/6e24004151d795c2e747ebe25280568eb23c0155)
> I am not sure I understand the thing about dist-packages vs site-packages compared to debian vs everyone else. A commit was made to generalize this a while ago [0d6aa71](https://github.com/gnuradio/gnuradio/commit/0d6aa71a114a2d2c6be95bc59c45a621bf053c82)
> But this was reverted shortly after again in [9f5e65d](https://github.com/gnuradio/gnuradio/commit/9f5e65d8ae9608acf7526095787d37c95f8fd543)
> 
> I am not exactly sure what the issue was with it that broke the CMAKE_INSTALL_PREFIX.
> 
> [53ee662](https://github.com/gnuradio/gnuradio/commit/53ee6625782176915756a89bfc625d62d8bbad67)
> Your last changes are related the following pull request. Did you consider it?
> #2293



> [6e24004](https://github.com/gnuradio/gnuradio/commit/6e24004151d795c2e747ebe25280568eb23c0155)
> I am not sure I understand the thing about dist-packages vs site-packages compared to debian vs everyone else. A commit was made to generalize this a while ago [0d6aa71](https://github.com/gnuradio/gnuradio/commit/0d6aa71a114a2d2c6be95bc59c45a621bf053c82)
> But this was reverted shortly after again in [9f5e65d](https://github.com/gnuradio/gnuradio/commit/9f5e65d8ae9608acf7526095787d37c95f8fd543)
> 
> I am not exactly sure what the issue was with it that broke the CMAKE_INSTALL_PREFIX.
> 
> [53ee662](https://github.com/gnuradio/gnuradio/commit/53ee6625782176915756a89bfc625d62d8bbad67)
> Your last changes are related the following pull request. Did you consider it?
> #2293

I did not see that one but it looks to be the identical problem with NetBSD

**Comment by noc0lour** [comment]
@DianeBruce The qwt patch looks kinda reasonable to me, can you test if vocoder/gsm works with the newest patches/master ?

**Comment by noc0lour** [comment]
The python thingy annoys me as well, current approach is not only wrong for FreeBSD but basically for everyone else aside from debian/ubuntu.

File: cmake/Modules/FindQwt.cmake [hunk 2]
```diff
@@ -29,8 +30,9 @@ find_path(QWT_INCLUDE_DIRS
 )
 
 find_library (QWT_LIBRARIES
-  NAMES ${PC_QWT_LIBRARIES} qwt6-${QWT_QT_VERSION} qwt-${QWT_QT_VERSION}
+  NAMES ${PC_QWT_LIBRARIES} qwt6 qwt6-${QWT_QT_VERSION} qwt-${QWT_QT_VERSION}
```
Comment (noc0lour) [inline cmake/Modules/FindQwt.cmake]: We removed that on purpuose, until all the qwt6 libraries linked to qt4 faded away. You should be able to override settings here with a cmake switch if you know where your qwt library is.

File: cmake/Modules/FindQwt.cmake [hunk 3]
```diff
@@ -15,6 +15,7 @@ find_path(QWT_INCLUDE_DIRS
   ${CMAKE_INSTALL_PREFIX}/include/qwt
   ${CMAKE_PREFIX_PATH}/include/qwt
   PATHS
+  /usr/local/include/qt5/qwt6
```
Comment (noc0lour) [inline cmake/Modules/FindQwt.cmake]: afaik this is overly specificy for systems where qt5 & qt4 are still alive.

**Comment by noc0lour** [comment]
Can you reopen if there is still something about the qwt we can merge or a more general python fix which will apply to all of our use-cases?
